packages/core/src/components/helpcenter-panel/helpcenter-panel.scss

```diff
@@ -96,7 +96,18 @@
   right: -640px;
```

Comment:
deze `-640px` is van belang bij het animeren. Door de breedte aan te passen van het paneel en de right op `-640px` te laten is de animatie op elke viewport iets anders.

Reply:
Dat heb ik gemist.